runelite-client/src/main/java/net/runelite/client/plugins/dpscounter/DpsOverlay.java

```diff
@@ -95,15 +95,14 @@ public void onMouseOver()
 	public Dimension render(Graphics2D graphics)
 	{
 		Map<String, DpsMember> dpsMembers = dpsCounterPlugin.getMembers();
-		if (dpsMembers.isEmpty())
-		{
-			return null;
-		}
-
 		boolean inParty = !partyService.getMembers().isEmpty();
 		boolean showDamage = dpsConfig.showDamage();
 		DpsMember total = dpsCounterPlugin.getTotal();
 		boolean paused = total.isPaused();
+		if (dpsMembers.isEmpty() && total.getStart() == total.getEnd())
```

Comment:
This is probably wrong as its a reference comparison of Instants. What are you trying to do here? Maybe just check && total.getDamage() == 0 instead?

Reply:
If 0 dmg is dealt that would still start total's timer which would mean that the timer is still started but not visible to the player which is what I was aiming to fix.

Reply:
This seems fine then I guess. Maybe make an `isReset` in DpsMember then because this is currently relying on an implementation detail of reset() using the same instant for start/end which is very subtle. And please update the commit message to describe what behavior you are trying to change.